node/core/dispute-coordinator/src/real/mod.rs

```diff
@@ -159,7 +159,10 @@ impl DisputeCoordinatorSubsystem {
 		Context: SubsystemContext<Message = DisputeCoordinatorMessage>,
 		B: Backend + 'static,
 	{
-		let res = self.initialize(&mut ctx, backend, &*clock).await?;
+		let res = match self.initialize(&mut ctx, backend, &*clock).await {
+			Ok(res) => res,
+			Err(err) => return Err(err),
+		};
```

Comment:
```suggestion
		let res = self.initialize(&mut ctx, backend, &*clock).await?;
```

Probably some debugging leftover? :D

Reply:
Ah yes, I made a mess with my debugging 😁